requirements.txt

```diff
@@ -1,3 +1,4 @@
+netaddr
```

Comment:
I opened up this issue after noticing dependencies are no pinned: https://github.com/openwisp/netengine/issues/58, let's do it after this.